dll/ntdll/ldr/ldrinit.c

```diff
@@ -1538,7 +1538,9 @@ VOID
 NTAPI
 LdrpValidateImageForMp(IN PLDR_DATA_TABLE_ENTRY LdrDataTableEntry)
 {
-    UNIMPLEMENTED;
+    DPRINT("LdrpValidateImageForMp is unimplemented\n");
```

Comment:
Alternatively, [`UNIMPLEMENTED_ONCE;`](https://git.reactos.org/?p=reactos.git;a=blob;f=sdk/include/reactos/debug.h;hb=56988ffbf8fff9e7d20130c52ddf7c9e972df658#l122) : emits the warning just once and doesn't spam.

Reply:
Do you know what this is supposed to do?

Reply:
> that is per-process so it does nag quite a bit.

Oh, indeed, so let's keep DPRINT.

Reply:
> Do you know what this is supposed to do with the LockPrefixTable?

The table contains the offsets in the PE image where all the instructions lock prefixes are present in it, and replaces these with nops (as an "optimization" on UP).